src/sparseml/optim/manager.py

```diff
@@ -118,19 +131,87 @@ def compose_staged(
         if not isinstance(additional_recipe, BaseManager):
             additional_recipe = cls.from_yaml(additional_recipe)
 
-        if isinstance(base_recipe.modifiers, OrderedDict):
-            raise ValueError(
-                "non-staged recipes not yet supported for Manager.compose_staged "
-                "found base_recipe with non_staged modifiers"
+        # Both base_recipe and additional_recipe are non-staged_recipes
```

Comment:
as discussed, let's default the `base_recipe` stage name to `pre_{first_additional_stage_name}` and `additional_recipe` stage name to `stage_{num_base_stages + 1}`